.github/workflows/woodwork_main_airflow_performance_tests.yaml

```diff
@@ -34,11 +34,9 @@ jobs:
           --data-raw '{
             "conf": {
                   "description": null,
-                  "lg_branch": "lg_baton_airflow",
```

Comment:
no need to specify the branch?

Reply:
probably defaults to `main`?

Reply:
exactly, default is main and that is set in Airflow. That branch was added while we were still in the development phase.